package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "deon-api-client",
-  "version": "0.1.0",
+  "name": "@deondigital/api-client",
+  "version": "1.0.0",
   "description": "REST client for Deon Digital CSL service",
-  "repository": "https://github.com/deondigital/deon-api-client/",
+  "repository": "https://github.com/deondigital/api-client/",
```

Comment:
In the `firetruck` module there is a "type: git" field here.  Maybe we should have one here too for consistency.